CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [v0.6.0](https://github.com/feast-dev/feast/tree/v0.6.0) (2020-MM-DD)
+
+[Full Changelog](https://github.com/feast-dev/feast/compare/v0.5.1...v0.6.0)
+
+**Implemented enhancements:**
+- Add support for staging files on S3 [\#706](https://github.com/feast-dev/feast/pull/769) ([jmelinav](https://github.com/jmelinav))
```

Comment:
You don't need to add this yet. We will add it when cutting the release.

Reply:
Oh, I thought this is what you meant by "add a note in the release. I will revert this change."